src/libcollections/slice.rs

```diff
@@ -860,6 +866,9 @@ pub trait SliceConcatExt<T: ?Sized> {
     fn connect(&self, sep: &T) -> Self::Output;
 }
 
+#[unstable(feature = "slice_concat_ext",
+           reason = "trait should not have to exist",
+           issue = "27747")]
```

Comment:
This is actually a bit of an interesting declaration because it's basically stable right now that slices have these methods, it's just unstable to import them or use UFCS via the trait. Not entirely sure what that means about the stability of the impl block itself, but it's fine either way